docs/user/sbt.rst

```diff
@@ -98,7 +98,14 @@ Garbage collectors
 
    Garbage collector that allocates things without ever freeing them. Useful
    for short-running command-line applications or applications where garbage
-   collections pauses are not acceptable.
+   collections pauses are not acceptable.`
+
+2. **immix.**
+
+    Immix is a mostly precise mark-region collector based on the paper:
+    `Immix: A Mark-Region Garbage Collector with Space Efficiency,
+     Fast Collection, and Mutator Performance
+    <http://www.cs.utexas.edu/users/speedway/DaCapo/papers/immix-pldi-2008.pdf>`_.
```

Comment:
We can't really expect our end-users to read papers to understand what we did. Contributors maybe, but not  the end-users.